xml/System.IdentityModel.Selectors/SecurityTokenProvider.xml

```diff
@@ -428,11 +428,20 @@
         <Parameter Name="token" Type="System.IdentityModel.Tokens.SecurityToken" Index="1" FrameworkAlternate="dotnet-plat-ext-3.1;dotnet-plat-ext-5.0" />
       </Parameters>
       <Docs>
-        <param name="timeout">To be added.</param>
-        <param name="token">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="timeout">A <see cref="T:System.TimeSpan" /> that specifies the timeout value for the message that cancels the security token.</param>
```

Comment:
The type will already show up next to the param, so this can be simplified:
```suggestion
        <param name="timeout">The timeout for the message that cancels the security token.</param>
```

Reply:
I was copying this from the documentation for the sync version of the api. Is it better to keep things consistent or to use the newer way of writing this?

Reply:
The latter. This repo's wiki mentions this guideline for params:

https://github.com/dotnet/dotnet-api-docs/wiki/Parameters

Reply:
I'll go back and fix the other instances in this document to keep at least this document self consistent.